enatega-multivendor-app/src/components/Main/TopPicks/TopPicks.js

```diff
@@ -1,70 +1,68 @@
 import styles from '../OrderAgain/styles'
 import TextDefault from '../../Text/TextDefault/TextDefault'
-import { alignment } from '../../../utils/alignment'
 import ThemeContext from '../../../ui/ThemeContext/ThemeContext'
 import { theme } from '../../../utils/themeColors'
-import { scale } from '../../../utils/scaling'
 import { useTranslation } from 'react-i18next'
 import NewRestaurantCard from '../RestaurantCard/NewRestaurantCard'
-import { gql, useQuery } from '@apollo/client'
-import { mostOrderdRestaurantsInfo } from '../../../apollo/queries'
-import { LocationContext } from '../../../context/Location'
+import { Placeholder, PlaceholderLine, Fade } from 'rn-placeholder'
 
-const MOST_ORDERED_RESTAURANTS = gql`
-  ${mostOrderdRestaurantsInfo}
-`
 function TopPicks(props) {
-  const { location } = useContext(LocationContext)
-
-  const { loading, error, data } = useQuery(MOST_ORDERED_RESTAURANTS, {
-    variables: {
-      latitude: location.latitude,
-      longitude: location.longitude
-    }
-  })
-
   const { t } = useTranslation()
   const themeContext = useContext(ThemeContext)
   const currentTheme = theme[themeContext.ThemeValue]
 
-  if (loading) return <Text style={styles().margin}>Loading...</Text>
-  if (error) return <Text style={styles().margin}>Error: {error.message}</Text>
+  function loadingScreen() {
+    return (
+      <View style={styles(currentTheme).screenBackground}>
+        <Placeholder
+          Animation={props => (
+            <Fade
+              {...props}
+              style={styles(currentTheme).placeHolderFadeColor}
+              duration={600}
+            />
+          )}
+          style={styles(currentTheme).placeHolderContainer}>
+          <PlaceholderLine style={styles().height200} />
+          <PlaceholderLine />
+        </Placeholder>
+      </View>
+    )
+  }
+
+  if (props?.loading) return loadingScreen()
+  if (props?.error)
+    return <Text style={styles().margin}>Error: {props?.error?.message}</Text>
 
-  // console.log(isLoggedIn, loading, error, data)
   return (
     <View style={styles().topPicksSec}>
-     
-        <TextDefault
-          numberOfLines={1}
-          textColor={currentTheme.fontFourthColor}
-          bolder
-          H4>
-          Top Picks for you
-        </TextDefault>
-        <TextDefault
-          Normal
-          textColor={currentTheme.secondaryText}
-          style={
-            styles().ItemDescription
-            }>
-          Most ordered right now.
-        </TextDefault>
-        <FlatList
-          style={styles().offerScroll}
-          showsVerticalScrollIndicator={false}
-          showsHorizontalScrollIndicator={false}
-          horizontal={true}
-          data={data?.mostOrderedRestaurants}
-          keyExtractor={item => item._id}
-          renderItem={({ item }) => {
-            return <NewRestaurantCard {...item} />
-          }}
-        />
-      </View>
-    
+      <TextDefault
```

Comment:
Order it again component and top picks have almost same content, it's better if we merge them and make one generic component for this.